OpenDreamClient/Interface/DreamInterfaceManager.cs

```diff
@@ -535,6 +535,8 @@ internal sealed class DreamInterfaceManager : IDreamInterfaceManager {
                 // But also have it here in case a local winget ever wants it
                 case "hwmode":
                     return "true";
+                case "windows":
+                    return string.Join(';', Windows.Keys);
```

Comment:
`winget(usr, null, "panes")` is also a thing which implies this shouldn't be returning panes (although I haven't tested this in BYOND). `Windows` also contains panes.

Reply:
Fixed